lang/csharp/src/apache/main/Avro.main.csproj

```diff
@@ -56,7 +56,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Hey @iemejia,

We actually don't want to update this dependency in this library. There's an extended description of the problem [here](https://github.com/apache/avro/pull/981#discussion_r525692847).

The [docs here](https://dependabot.com/docs/config-file/) show how I can configure `ignored_updates` in the `dependabot.yml`. Is there a way I can test out locally that the configuration change is working?

Reply:
No idea, maybe this would appear in https://github.com/apache/avro/network/updates of your fork but not sure. Maybe enabling this in your fork will help you see it somewhere, notice that if you enable this in your fork you will be swamped by PR requests.

The syntax for github's dependabot is not the same of the one on your link, it is v2 and I have not found the docs for that, the closest I found was VS Code helping me with suggestions (It uses this schema https://json.schemastore.org/dependabot-2.0).